components/automate-ui/src/app/pages/user-details/user-details.component.html

```diff
@@ -2,25 +2,25 @@
 <app-profile-sidebar *ngIf="!isAdminView"> </app-profile-sidebar>
 
 <div class="container">
-  <main *ngIf="isAdminView || user.id">
+  <main *ngIf="isAdminView || user?.id">
     <app-delete-object-modal
-      [visible]="modalVisible"
-      objectNoun="user"
-      [objectName]="user.name"
-      (close)="closeDeleteConfirmationModal()"
-      (deleteClicked)="deleteUser()"
-      objectAction="Delete">
+    [visible]="modalVisible"
+    objectNoun="user"
+    [objectName]="user?.name"
+    (close)="closeDeleteConfirmationModal()"
+    (deleteClicked)="deleteUser()"
+    objectAction="Delete">
```

Comment:
Probably an auto-format caused this change; would be preferable to revert to keep the indents for readability, though.